app/services/update_account_service.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class UpdateAccountService
```

Comment:
@VladislavSokov please explain why it's needed and why it's important to implement now?

Reply:
And additionally, with this approach, we can edit a kid's email, regardless of notification settings.